environments/stage/extensions.json

```diff
@@ -25,11 +25,5 @@
   },
   {
     "source": "https://raw.githubusercontent.com/kyma-project/eventing-manager/main/config/ui-extensions/subscriptions/subscriptions.yaml"
-  },
-  {
-    "source": "https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/config/busola/logpipeline_busola_extension_cm.yaml"
-  },
-  {
-    "source": "https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/config/busola/tracepipeline_busola_extension_cm.yaml"
```

Comment:
why are you removing logpipeline and tracepipeline extensions? are they part of extensibility?